docs/Downloaders/qBittorrent/Tips/How-to-run-the-unRaid-mover-for-qBittorrent.md

```diff
@@ -214,16 +189,27 @@ Choose your method (select a tab) and copy/paste the script in the new window th
 
     ``` bash
         #!/bin/bash
+
+        source /mnt/user/data/scripts/.venv/bin/activate
+
         /usr/local/emhttp/plugins/dynamix/scripts/notify -s "qBittorrent Mover" -d "qBittorrent Mover starting @ `date +%H:%M:%S`."
         echo "executing script to pause torrents and run mover."
-        /mnt/user/data/scripts/.venv/bin/python3 /mnt/user/data/scripts/mover.py
+        python3 /mnt/user/data/scripts/mover.py --host "localhost:8080" --user "your_user" --password "your_password" --days_from 0 --days_to 2
         echo "qbittorrent-mover completed and resumed all paused torrents."
         /usr/local/emhttp/plugins/dynamix/scripts/notify -s "qBittorrent Mover" -d "qBittorrent Mover completed @ `date +%H:%M:%S`."
+
+        deactivate
```

Comment:
Changing this to activate the venv for one command isn't really needed, I'm not sure why you're suggesting to make this change?

Reply:
Yea, I think you and I discussed this change initially when I redid the mover script pages while adding Deluge's mover...

Reply:
fair enough